Makefile

```diff
@@ -19,3 +19,9 @@ local:
 
 start_all:
```

Comment:
Once the PR is merged I will be sure to update the wikis/README to show the new way to start all servers (both with and without ASR).